tests/kernel/mbox/mbox_api/src/test_mbox_api.c

```diff
@@ -419,7 +419,8 @@ static void tmbox_get(struct k_mbox *pmbox)
 			     NULL);
```

Comment:
Please put this with other code change and separate it from the coccinelle script change.